sources/academy/tutorials/apify_actors/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Tutorials on Apify Actors
+description: A collection of various Actor tutorials to aid you in your journey to becoming a master Actor developer.
+sidebar_position: 7
+category: tutorials
+slug: /apify-actors
+---
+
+# Apify Actors
+
+**Learn how to deploy your API project to the Apify platform.**
+
+---
+
+This tutorial shows you how to add your existing RapidAPI project to Apify, giving you access to managed hosting, data storage, and a broader user base through Apify Store while maintaining your RapidAPI presence.
+
+- [Adding your RapidAPI project to Apify](./adding_rapidapi_project.mdx)
```

Comment:
Only now I noticed it's a .mdx file. I think we use those just where it's absolutely necessary, otherwise we keep the files with the .md extension.

Reply:
I changed it to `.md`